src/App.ts

```diff
@@ -1,12 +1,42 @@
 import express from 'express';
 import path from 'path';
+import {User} from "./backend/Models";
+import bodyParser from 'body-parser';
+const mongoose = require('mongoose');
 
 const app = express();
 const port = 3000;
+app.use(bodyParser.json());
+
+// mongoose connection, connect to local database with db name "project", creates new database if "project" doesn't exist
+mongoose.connect('mongodb://localhost:27017/project', {useNewUrlParser: true});
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function() {
+    console.log('mongoose connected!');
+});
 
 app.use(express.static(path.join(__dirname, '..', 'public')));
+
+// TODO: will move to a different file after rebasing with backend apiRouter setup PR
+// Sample apis for User Schema:
+app.post('/user', (req, res) => {
+    User.create(req.body, (err: Error, user: any) => {
+        console.log(err, user);
+        res.send(user);
```

Comment:
Also need handle error here?

Reply:
How are we planning to deal with errors? Log and send back to frontend to display for user?